pages.html/pg2.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Contact Me</title>
+        <a href="pg1.html">Resume</a>
+        <h1>Contact Me</h1>
+<hr>
+        <label for="firstname">First Name</label><br>
+        <input required type="text" id="firstname" name="firstName"> <br><br>
+
+        <label for="lastname">Last Name </label><br>
+        <input required type="text" id="lastname" name="lastName"> <br><br>
+
+        <label for="phonenumber">Phone Number</label><br>
+        <input type="text" id="phonenumber" name="phoneNumber"><br>
+        
+        <label for="phonetype">Mobile</label>
+        <input type="radio" id="phonetype" name="phoneType"> 
+        <label for="phonetype">Landline</label>
+        <input type="radio" id="phonetype" name="phoneType"> <br><br>
+
+        <label for="email">Email</label><br>
+        <input required type="text" id="email" name="email" maxlength="50"><br><br>
+        
+
+        <label for="interest">Skills of Interest</label> 
+            <select>
+                <option value="miscrosoft">Microsoft</option>
+                <option value="interpersonal">Interpersonal</option>
+                <option value="customerservice">Customer Service</option>
+            </select>
+        <br><br>
+
+
+        <label for="comments">Comments</label><br>
+        <textarea name="comments" cols="50" row="10" maxlength="300"></textarea> <br>
+
+        <button>Submit</button>
+
+    </body>
+</html>
```

Comment:
Missing How did you hear about us dropdown (-5)
-------------------------------------------------
80/100
Great work! My only recommendation is to make sure you do everything the lab asks for. You missed some points on silly mistakes.